.github/workflows/windows.yml

```diff
@@ -0,0 +1,34 @@
+name: windows
+
+on: 
+  push:            # run on push events
+    paths-ignore:  # but ignore everything in the docs subfolder 
+      - 'docs/**'
+    branches:
+      - '**'
+    tags:
+      - '*'
+  pull_request:    # run on pull requests
+    paths-ignore:  # but ignore everything in the docs subfolder 
+      - 'docs/**'
+  schedule:
```

Comment:
I'm going to remove this later.
The tests on the released version of SUMO should be only on push and PR.